recipes/qfilt/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: qfilt
+  version: 0.0.1
+
+source:
+  git_rev: 0.0.1
```

Comment:
@stevenweaver can you please use a tarball. github is able to give you a tarball to every single commit. This will enable reproducibility. Please also include a checksum, e.g. sha256